### DISCUSSION TITLE
Allow row to be read based on owner from another collection

### DISCUSSION THREAD

**Discussion by srowe0091** [pocketbase/pocketbase#5664] [Q&A] [answered]
Hello! I have this current table setup
![image](https://github.com/user-attachments/assets/b47589e2-ea22-4591-97fc-4b958b184a40)

and im trying to apply an API Rule that permits the user to be able to read the row of a card, if they are the owner of the storage.

for example, if `user 1` owns `storage 999` they should be able to read all the rows in `cards` where the `storageRelationId = 999` and if `user 2` tries to read `storage 999`, they can not because they are not the owner

**Comment by bmingles** [comment, marked as answer]
Have you tried 
`@request.auth.id = storageRelationId.ownerRelationId`?

**Reply by bmingles** [reply]
Or possibly
`@request.auth.id = storage.owner` depending on what your relation field names actually are in the UI

**Reply by bmingles** [reply]
I think you can use the name of the relation field (as displayed in the UI) to reference the id.

**Reply by srowe0091** [reply]
the first suggestion was it! I knew it had to be something very simple lol. thank you!
